cluster/cluster_impl/failover_cluster.go

```diff
@@ -44,3 +44,7 @@ func NewFailoverCluster() cluster.Cluster {
 func (cluster *failoverCluster) Join(directory cluster.Directory) protocol.Invoker {
 	return newFailoverClusterInvoker(directory)
 }
+
+func GetFailoverName() string {
```

Comment:
Pls move the const into directory "constant"

Reply:
Other cluster_impl is def private name because they not used out package, failover and zoneAware is the special, used in reference_config for select or default set, I want to put the two constant to new file common/constant/cluster.go. 
```go
const (
	FAILOVER_CLUSTER_NAME  = "failover"
	ZONEAWARE_CLUSTER_NAME = "zoneAware"
)
```
How about this?